app/jobs/rstuf/application_job.rb

```diff
@@ -0,0 +1,5 @@
+class Rstuf::ApplicationJob < ApplicationJob
+  before_enqueue do
+    throw :abort unless Rstuf.enabled?
+  end
```

Comment:
Nit, with only 3 subclasses, I'd put this as a one liner at the top of each job class. Do you expect more jobs than this?

Reply:
Potentially this base job could be used for specific RSTUF job config, like retries, queue setup, ... I thought it is worth to keep isolated. Also potentially whole `Rstuf` namespace can be off-boarded in separate gem.

Reply:
That seems fine. 👍